index.js

```diff
@@ -98,13 +113,15 @@ if (process.platform === 'win32') {
 // start the children
 children = [];
 cmds.forEach(function (cmd) {
-    if (process.platform != 'win32') {
-      cmd = "exec "+cmd;
+    if (process.platform === 'win32') {
+        cmd = cmd.replace(/'/g,"\"");
     }
     var child = spawn(sh,[shFlag,cmd], {
         cwd: process.cwd,
         env: process.env,
-        stdio: ['pipe', process.stdout, process.stderr]
+        stdio: ['pipe', process.stdout, process.stderr],
+        windowsVerbatimArguments: process.platform === 'win32',
+        detached: process.platform != 'win32'
```

Comment:
I don't see any output when I run commands now. Seems this line is the culprit. Why are we detaching the process?